src/fu-util.c

```diff
@@ -2587,7 +2620,7 @@ fu_util_prompt_warning_bkc(FuUtilPrivate *priv, FwupdDevice *dev, FwupdRelease *
 	g_string_append_printf(
 	    str,
 	    /* TRANSLATORS: %1 is the current device version number, and %2 is the
-	       command name, e.g. `fwupdmgr sync-bkc` */
+	       command name, e.g. `fwupdmgr sync` */
```

Comment:
Can you add sync-bkc as deprecated for a while so as to not break people's existing scripts too?

Reply:
Yup, that's what the `sync,sync-bkc` bit achieves.